pulpcore/pulpcore/app/viewsets/base.py

```diff
@@ -47,7 +47,7 @@ def _allows_filters(self, path, method):
             return False
 
         if hasattr(self.view, 'action'):
-            return self.view.action in ["list", "retrieve"]
+            return self.view.action in ["list"]
```

Comment:
I double checked and this does not, actually, change the output schema at all.  But we shouldn't have it there anyway.